yoti-sdk-api/src/main/java/com/yoti/api/client/identity/constraint/SourceConstraint.java

```diff
@@ -10,14 +10,16 @@
 
 public class SourceConstraint implements Constraint {
 
+    private static final String SOURCE_CONSTRAINT = "SOURCE";
```

Comment:
```
TYPE = "SOURCE"
```
the class is already named `SourceConstraint` so the `SOURCE` value name is relative to `SourceConstraint`.

`SourceConstraint.SOURCE_CONSTRAINT` is obviously not ideal, `SourceConstraint.TYPE` is much nicer.